### PR TITLE
Add Readme

### PR DESCRIPTION
**Pull request by sauercrowd**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+if err != nil {
+    log.Fatal("b", err)
+}
+
+connection, err := net.ListenUDP("udp4", address)
+if err != nil {
+	log.Fatal("a", err)
+}
+
+// our service can handle 128 packets per second
```
Comment (lmb) [inline README.md:30]: This comment is misleading: we don't rate limit to 128 packets in total.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+service in question won't be able to handle anymore:
+
+```go
+
+address, err := net.ResolveUDPAddr("udp4", ":9876")
+if err != nil {
+    log.Fatal("b", err)
+}
+
+connection, err := net.ListenUDP("udp4", address)
```
Comment (lmb) [inline README.md:25]: I think it's easier if you do `ListenPacket("udp4", "127.0.0.1:9876")` and then assert `connection.(*net.UDPConn)`

File: README.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+if err != nil {
+	log.Fatal("a", err)
+}
+
+// our service can handle 128 packets per second
+ppsPerSecond := 128
+rake, err := New(connection, ppsPerSecond)
+defer rake.Close()
```
Comment (lmb) [inline README.md:33]: Comment that the filter stays active even after close?

File: README.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+```
+
+That's all! The library now enforces rate limits on incoming packets, and it happens within the kernel.
+
+## Limitations
+- no IPv6 (we're working on adding it)
+- requires tweaking of optmem
+- not production ready
```
Comment (lmb) [inline README.md:41]:
```suggestion
- not tested in production
```

File: README.md (hunk 1 above)
Comment (sauercrowd) [reply README.md:30]: Good point

**Review by lmb — APPROVED** [review]

**Pull request merged**